test/integration/mount_test.go

```diff
@@ -46,7 +46,7 @@ func testMounting(t *testing.T) {
 	defer os.RemoveAll(tempDir)
 
 	mountCmd := fmt.Sprintf("mount %s:/mount-9p", tempDir)
-	cmd := minikubeRunner.RunDaemon(mountCmd)
+	cmd, _ := minikubeRunner.RunDaemon(mountCmd)
```

Comment:
Ditto

Reply:
This is necessary because the RunDaemon signature now returns a stdout reader to support the new blocking behavior of the "minikube dashboard" command.